src/ralph/models/xapi/video/contexts.py

```diff
@@ -32,17 +31,13 @@
 
 
 class VideoContextContextActivitiesCategory(BaseXapiActivity):
-    # noqa: D205, D415
-    """Pydantic model for video `context`.`contextActivities`.`category`
-    property.
+    """Pydantic model for video `context`.`contextActivities`.`category` property.
 
     Attributes:
         id (str): Consists of the value `https://w3id.org/xapi/video`.
-        definition (dict): see ProfileActivity.
     """
 
     id: Literal["https://w3id.org/xapi/video"] = "https://w3id.org/xapi/video"
-    definition: ProfileActivity
```

Comment:
You're right, it is not a mandatory field to have for validation. But we can keep it as optional, no? WDYT?